services/movie_session.py

```diff
@@ -42,3 +42,8 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
```

Comment:
The `delete_movie_session_by_id` function should include exception handling to manage cases where a movie session with the specified ID does not exist. Consider using a try-except block to catch `MovieSession.DoesNotExist` exceptions.